src/screens/Settings/components/OtherSettings/index.tsx

```diff
@@ -306,23 +321,23 @@ export default function OtherSettings({
               onChange={handleLauncherArgs}
             />
           </span>
+          <InfoBox text="infobox.help">
```

Comment:
this InfoBox code is duplicated, maybe you can store the element in a variable before the conditionals and then do `{infoBox}`